packages/@okta/vuepress-site/docs/reference/api/schemas/index.md

```diff
@@ -2411,34 +2431,62 @@ App User Profile schema properties have the following standard [JSON Schema Draf
 | type                           | type of property                           | `string`, `boolean`, `number`, `integer`, `array` | FALSE    | FALSE  | FALSE    |
 | enum                           | enumerated value of the property           | array                                             | TRUE     | TRUE   | FALSE    |
 | oneOf                          | non-empty array of valid JSON schemas      | array                                             | TRUE     | TRUE   | FALSE    |
+| format                         | identifies the type of data represented by the string | string                                            | TRUE     | TRUE   | FALSE    |
 
-##### Description Details
-
-* `enum`: The value of the property is limited to one of the values specified in the `enum` definition.  The list of values for `enum` has to be made up of unique elements.
-
-* `oneOf`: Okta only supports `oneOf` for specifying display names for an `enum`. Each schema has the following format:
-
- ```json
-{
-  "const": "enumValue",
-  "title": "display name"
-}
- ```
+##### Description details
 
-In case `enum` is used in conjunction with `oneOf`, the set of enumerated values and their order must be kept.
+- `enum`: The value of the property is limited to one of the values specified in the `enum` definition. The list of values for the `enum` must be made up of unique elements.
 
-```json
-{"enum": ["S","M","L","XL"],
-  "oneOf": [
-    {"const": "S", "title": "Small"},
-    {"const": "M", "title": "Medium"},
-    {"const": "L", "title": "Large"},
-    {"const": "XL", "title": "Extra Large"}
-  ]
-}
- ```
+- `oneOf`: Okta only supports `oneOf` for specifying display names for an `enum`. Each schema has the following format:
 
-`oneOf` is only supported in conjunction with `enum`, providing a mechanism to return a display name for the `enum` value.
+  ```json
+  {
+    "const": "enumValue",
+    "title": "display name"
+  }
+  ```
+
+  In case `enum` is used in conjunction with `oneOf`, the set of enumerated values and their order must be kept.
+
+  ```json
+  {"enum": ["S","M","L","XL"],
+    "oneOf": [
+      {"const": "S", "title": "Small"},
+      {"const": "M", "title": "Medium"},
+      {"const": "L", "title": "Large"},
+      {"const": "XL", "title": "Extra Large"}
+    ]
+  }
+  ```
+
+  `oneOf` is only supported in conjunction with `enum`, providing a mechanism to return a display name for the `enum` value.
+
+- `format`: Okta uses this keyword to identify the type of data represented by the string. The following attribute formats are supported: `uri`, `date-time`, `email`, `ref-id`, `encrypted`, `hashed`, `country-code`, `language-code`, `locale`, and `timezone`. See the following request that uses the format property:
+
+  ```bash
+  curl --location --request POST 'https://{$yourOktaDomain}/api/v1/meta/schemas/apps/0oa4cthzyqgFj8ISJ5d7/default' \
```

Comment:
Consider replacing `0oa4cthzyqgFj8ISJ5d7` with a placeholder like `${appId}`

Also, looks like `{$yourOktaDomain}` should be `${yourOktaDomain}`